linuxclientsetup/scripts/client-config

```diff
@@ -293,6 +293,50 @@ if [[ -L /usr/lib/firefox/dictionaries ]]; then
 	ln -s /usr/share/hunspell/"$machineLang".aff /usr/lib/firefox/dictionaries/"$machineLang".aff
 fi
 
+#Lockdown preferences
+firefox_version=`firefox --version | sed 's/^[^0-9]*//'`
+	#Proxy server
+	if [[ $PROXYSERVER ]]; then
+		cat << EOF > /var/lib/karoshi/firefox/mozilla.cfg 
+//
+pref("network.proxy.http", "$PROXYSERVER.$DNSSUFFIX");
+pref("network.proxy.http_port", 3128);
+pref("network.proxy.ssl", "$PROXYSERVER.$DNSSUFFIX");
+pref("network.proxy.ssl_port", 3128);
+pref("network.proxy.ftp", "$PROXYSERVER.$DNSSUFFIX");
+pref("network.proxy.ftp_port", 3128);
+pref("network.proxy.socks", "$PROXYSERVER.$DNSSUFFIX");
+pref("network.proxy.socks_port", 3128);
+pref("network.proxy.gopher", "$PROXYSERVER.$DNSSUFFIX");
+pref("network.proxy.gopher_port", 3128);
+pref("network.proxy.type", 1);
+pref("network.proxy.no_proxies_on", "localhost, 127.0.0.0/8, 192.168.0.0/16, 172.16.0.0/12, 10.0.0.0/8, $DNSSUFFIX, $PDC");
+lockPref("browser.startup.homepage", "$PROXYSERVER.$DNSSUFFIX");
+EOF
+	else
+		#For no proxy, we want the homepage to be about:home
+		cat << FOE > /var/lib/karoshi/firefox/mozilla.cfg
+lockPref("browser.startup.homepage", "about:home");
+FOE
+	fi
+
+	#Remove updated messages and extension checking
+	cat << FOE > /var/lib/karoshi/firefox/mozilla.cfg
+pref("browser.startup.homepage_override.mstone", "$firefox_version");
+pref("extensions.lastAppVersion", "$firefox_version");
+pref("extensions.lastPlatformVersion", "$firefox_version");
+FOE
+
+	#Setup Kerberos
+	cat << FOE > /var/lib/karoshi/firefox/mozilla.cfg
+lockPref("network.negotiate-auth.trusted-uris", "$DNSSUFFIX");
+FOE
+
+	#Set spellchecker language to machine language
+	cat << FOE > /var/lib/karoshi/firefox/mozilla.cfg
+pref("spellchecker.dictionary", "$machineLang");
+FOE
+
```

Comment:
You are using `>` to redirect output into mozilla.cfg, but this will overwrite the contents every time you run it, so the file will only ever contain the dictionary line. Use `>>`.